pyproject.toml

```diff
@@ -0,0 +1,185 @@
+[tool.poetry]
```

Comment:
If the package metadata is being moved into pyproject.toml, it should use PEP 621, not any legacy formats.